tests/testthat/test-workflows.R

```diff
@@ -20,22 +19,22 @@ test_that("can get summary of a workflow", {
                list(reports = list(
                  list(name = "other")
                ),
-               ref = NULL,
-               missing_dependencies = list(
-                 other = list()
-               )))
+                    ref = NULL,
```

Comment:
I'm just looking at this stying discrepancy and what IntelliJ is doing here is lining up the members of the second level list - `reports` is the first item, `ref` is the second, so `reports` and `ref` are vertically aligned. This does seem to be objectively "correct". 

Spent a while digging around with RStudio to figure out how it controls project level formatting and seems it is limited to 4 options which are declared in the `.Rproj` file: https://github.com/vimc/orderly.server/blob/master/orderly.server.Rproj#L7

So then I had a little go with RStudio (applying `ctrl + i` as you suggested) and it did not revert my changes here, so it seems my RStudio is in agreement with Intellij on this issue, and indeed I haven't seen another discrepancy so I think for now we can be reasonably sure that both IDEs agree on formatting.

Reply:
Yeah this block looks good, my RStudio is also happy with it. Does your RStudio not change line 32 - 35? I see IntelliJ it's aligning with `reports`, RStudio just putting aligning with the closing reports parenthesis :(. Maybe we should switch to
```
list(
  reports = list(
    list(name = "use_dependency")
  ),
  ref = NULL,
  missing_dependencies = list(
    use_dependency = list("other")
)))
```
to avoid noise? I think our autoformatters will agree on that? Same for block starting 186, 201, 220, 235, 259. Otherwise dw I can deal with

Reply:
That sounds like a good solution, will re-format all those blocks, we want to be able to auto-format in either IDE :)

Reply:
does the indentation work for you now @r-ash ?